src/officehours_ui/templates/base.html

```diff
@@ -60,7 +60,7 @@
                   </a>
                   <form action="{% url 'oidc_logout' %}" method="post">
                     {% csrf_token %}
-                    <input type="submit" value="Logout" style="background-color: white" class="dropdown-item">
+                    <input type="submit" value="Logout" style="background-color: white" class="dropdown-item" onclick="sendLogoutSignal()">
```

Comment:
Rather than having an onclick inline, it would probably be better to keep the code down in the script just where the sendLogutSignal is have an event listener for this. Give this an id like "logout-button". 

Then later

```
      const logoutButton = document.querySelector('#logout-button');
      logoutButton.addEventListener('click', () => {
        window.localStorage.setItem("loggedIn", "false");
      });
```